lib/features/preview-support/PreviewSupport.js

```diff
@@ -204,12 +190,12 @@ PreviewSupport.prototype._cloneMarker = function(gfx, marker, markerType, classN
 
     this._clonedMarkers[ markerId ] = clonedMarker;
 
-    var defs = domQuery('defs', this._canvas._svg);
+    var defs = domQuery(':scope > defs', parentGfx);

```

Comment:
What is the support for :scope within CSS?

Reply:
Everything except IE11

![image](https://github.com/bpmn-io/diagram-js/assets/21984219/bb3a077e-1f97-4106-b5f5-c7e863922dc7)